### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by kizombaciao** [Lightning-AI/dl-fundamentals#6] [Lecture videos and quizzes]
Just wondering, if there might be keyboard shortcuts to control the video.  Like going back 5 seconds.

**Comment by g-i-o-r-g-i-o** [comment]
something like youtube at least

**Comment by rasbt** [comment]
Good point. And the answer is yes, there are keyboard shortcuts to make the watching experience more convenient. You can use the arrow keys to scrub forward (and backward) 5 seconds. 

(And the up & down arrow keys adjust the volume.)
